client/src/components/NearestHospital/CheckSymptomsBox.js

```diff
@@ -45,14 +43,26 @@ const SymptomsLink = styled(NavLink)`
 `;
 
 const CheckSymptoms = (props) => {
+  const { t } = useTranslation();
+
   return (
     <CheckSymptomsBox>
-      <p>
-        Evaluate your health and receive instructions and recommendations about{" "}
-        <span>COVID-19</span>
-      </p>
-      <div style={{ flexBasis: "46%" }}>
-        <SymptomsLink to="/symptoms-check">Check Symptoms</SymptomsLink>
+      <div>
+        <p>
+          <span>{t("nearestHsp.hlthEval")}</span>
```

Comment:
hlth to health?

Reply:
This component will be developed in the future with another PR/feature branch.   But I will update it.

Reply:
done